### PR TITLE
Add lodash and security rules

### PR DESCRIPTION
**Pull request by DavidTanner**
security

Some of these were set up in the original `eslint-plugin-node` setup.

### CODE REVIEW

**Review by IevgenRagulin — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -32,6 +32,8 @@
     "@typescript-eslint/parser": "^4.29.3",
     "eslint-plugin-jest": "^24.4.0",
     "eslint-plugin-jsdoc": "^36.0.8",
-    "eslint-plugin-prefer-arrow": "^1.2.3"
+    "eslint-plugin-lodash": "^7.3.0",
+    "eslint-plugin-prefer-arrow": "^1.2.3",
+    "eslint-plugin-security": "^1.4.0"
```
Comment (smolster) [inline package.json:37]: It looks like this repo hasn't been updated in 3 years, and seems like it isn't being maintained. Maybe we should avoid using it for that reason?

Curious what you think. FWIW, it seems like CodeQL would detect at least some of the problems that this config claims to detect.

File: lib/javascript.js [hunk 2]
```diff
@@ -1,6 +1,14 @@
...
+  plugins: [
+    'prefer-arrow',
+    'jsdoc',
+    'security',
+    'lodash',
+  ],
   extends: [
     'eslint:recommended',
+    'plugin:security/recommended',
+    'plugin:lodash/recommended',
```
Comment (smolster) [inline lib/javascript.js:11]: The recommended config seems like [it's super opinionated](https://github.com/wix/eslint-plugin-lodash/blob/master/src/index.js#L8-L51). According to the docs, it includes _all_ of the rules.

[These rules](https://github.com/wix/eslint-plugin-lodash#preference-over-native) in particular seem like they might cause problems in a lot of projects, esp. those that don't use lodash at all.

What do you think?

File: package.json (hunk 1 above)
Comment (DavidTanner) [reply package.json:37]: It does seem the rules are not very solid, and old.  Not sure if it needs to be kept up to date very much.

File: lib/javascript.js (hunk 2 above)
Comment (DavidTanner) [reply lib/javascript.js:11]: I thought it would be nice, but then I read the rules, and don't see a reason to add it.

**Pull request closed without merging**